### DISCUSSION TITLE
How can I link with a google auth account after an email with password was registered?

### DISCUSSION THREAD

**Discussion by etriebe** [MoonriseSoftwareCalifornia/AspNetCore.Identity.CosmosDb#29] [General]
Many sites will allow you to authenticate with multiple authentication types: email/password and gmail/outlook if the email address matches. Is there a way to do this?

**Comment by etriebe** [comment]
@toiyabe is this supported? Should it be part of the package or something I need to build client side.

**Reply by toiyabe** [reply]
Hi Eric,

Just so I understand, a user has created an account with an email and password, now they want to associate this account with a Google or Microsoft account right?

If so, then after setting up an [external provider](https://learn.microsoft.com/en-us/aspnet/web-api/overview/security/external-authentication-services), the ASP.NET Identity UI has a "Manage" account area that allows a user to login, then manage their account's external logins.

This part of the UI won't appear if no external providers are established.

It's been a while since I did that, but I know that capability is there.

-Eric

**Reply by etriebe** [reply]
Ah, I missed that manage link in the login partial page. I've added it to my existing chrome now. 

One other thing, is there a way for me to modify the project name that is referenced in the navbar-brand link? I'd like to change the text in there without needing to rename my project.
